BinariesAPI.md

```diff
@@ -0,0 +1,107 @@
+# JULEP BinariesAPI
+
+- **Title:** A structured HTTP API for getting Julia binaries
+- **Authors:** Eric Davies <<iamed2@gmail.com>>
+- **Created:** June 16, 2017
+- **Status:** proposed
+
+## Abstract
+
+The URLs for fetching Julia binaries of various versions have changed a few times across
```

Comment:
I'll note that I've need to update Playground.jl a few time for this. Not a huge issue, but it would be nice if there was a stable url schema.